_posts/francois_gerthoffert/2019-03-03-ZenCrepes-GitHub-Agile-Analytics-made-easy.md

```diff
@@ -0,0 +1,301 @@
+
+Finally, the bottom of the view contains issues and points repartition by assignee, milestone, and label.
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/scrum-repartition.png" />
+    <figcaption>Repartition by assignee, milestone, and label</figcaption>
+</figure>
+
+All of those being available in the same view, it’s easy to scroll up and down between the various elements.
+
+# Manage Labels & Milestones
+
+As you might have figured out, it is challenging to manage labels and milestones across multiple organizations and repositories in GitHub.
+
+Those two views have been created to assist in the bulk creation, modification or deletion of labels and milestones.
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/labels-view.png" />
+    <figcaption>Labels view</figcaption>
+</figure>
+
+In the above screenshot, the “bug”, “invalid”, “help wanted” label have different labels colors. “Help wanted” in particular has only 1 repository with a different color.
+
+<figure style="width: 320px">
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/labels-colors.png"/>
+    <figcaption>Colors repartitions for “help wanted”</figcaption>
+</figure>
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/labels-colors-pick.png" />
+    <figcaption>Picking a color for the label</figcaption>
+</figure>
+
+Users can then update the color and bulk push the change to all repositories at once. Users can also push this label to additional repositories by clicking on the small folder icon on the left side.
+
+# Safeguards
+
+The challenge for ZenCrepes when operating with GitHub data is that there are no available backups. ZenCrepes is, therefore, doing its best to inform users about the bulk changes it’s about to perform.
+
+Before any bulk update, ZenCrepes fetches the latest version of the node and will “refuse” to push the bulk change if any of the nodes was modified since the last time the node was loaded locally. This prevents overwriting changes by mistake.
+
+Finally, ZenCrepes will also verify the user’s permission against the repository. Of course, I cannot update the label’s color into one of Microsoft public repositories. 
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/review-changes.png" />
+    <figcaption>Review changes before pushing to GitHub</figcaption>
+</figure>
+
+# Exploring
+
+When public GitHub repositories are configured to authorize fetching of data (most are), ZenCrepes also allows its users to add external repositories and organizations. Doing so has, of course, some limitations, such as story points or agile board which require a specific label format. 
+
+## Microsoft Issues
+
+Microsoft made the headlines sometime back when it became one of the most active Open Source projects on GitHub or when it actually acquired GitHub. But have you been curious to dive a bit into how Microsoft operates its teams?
```

Comment:
Perhaps you could link to this article (or similar)?
[https://visualstudiomagazine.com/articles/2018/10/17/octoverse-2018.aspx](https://visualstudiomagazine.com/articles/2018/10/17/octoverse-2018.aspx)